.gitignore

```
@@ -5,4 +5,3 @@ target/
 
 # TypeScript builds
 dist/js/*.js
```

```suggestion
dist/js/*.js
dist/js/*.map
```

deleted the wrong thing, whoops